src/makeCalculator.js

```diff
@@ -37,7 +37,39 @@
  * @return {object}
  */
 function makeCalculator() {
-  // write code here
-}
+  const object = {
+    result: 0,
+
+    operate(callback, number) {
+      callback(number);
+
+      return this;
+    },
+
+    add(number) {
+      object.result += number;
```

Comment:
Inside the methods 'add', 'subtract', 'multiply', 'divide', 'reset', you are referring to the 'object' explicitly. A better way to do this would be using 'this' keyword, which refers to the object that is executing the current method.

Reply:
I cant use 'this' because I used callback in 'operate' method, so 'this' in my methods uses callback's 'this', not object. I tried to use 'this', but got undefined instead of an object. Actually, I found a different way to solve this problem - use .bind(calculator) in tests, but I think it is a wrong way :)

Reply:
Still not fixed